test/unit/solver/test_lu_decomposition_in_place_policy.hpp

```diff
@@ -198,7 +218,7 @@ void testExtremeValueInitialization(std::size_t number_of_blocks, double initial
   CopyToHost<SparseMatrixPolicy>(ALU);
 
   check_results<double, SparseMatrixPolicy>(
-      A, ALU, [&](const double a, const double b) -> void { EXPECT_NEAR(a, b, 1.0e-09); });
+      A, ALU, [&](const double a, const double b) -> void { EXPECT_NEAR(a, b, 1.0e-14); });
```

Comment:
love this

Reply:
Looks like Windows does not like this. Revert it back.